src/core/src/CoreMain.py

```diff
@@ -101,6 +101,9 @@ def __init__(self, argv):
                     patch_installer.mark_installation_completed()
                     overall_patch_installation_operation_successful = True
                 self.update_patch_substatus_if_pending(patch_operation_requested, overall_patch_installation_operation_successful, patch_assessment_successful, configure_patching_successful, status_handler, composite_logger)
+            truncated_packages_removed = status_handler.get_assessment_truncated_removed()
+            if not len(truncated_packages_removed) == 0:
+                self.composite_logger.log_debug("Assessment truncated removed packages : {0}".format(truncated_packages_removed))
```

Comment:
Encapsulate this logic into status_handler inside a function called status_handler.log_truncated_removed_packages_if_any()

This may just be assessment now, but can also capture installation later.

We need to avoid leaking knowledge of things as much as possible outside respective classes.

Reply:
Also code coverage says not covered - re-check when this change is made.